superset-frontend/stylesheets/less/cosmo/variables.less

```diff
@@ -220,7 +220,7 @@
 );
 
 // ** `.form-group` margin
-@form-group-margin-bottom: 10px;
+@form-group-margin-bottom: 16px;
```

Comment:
moving towards the `8px` system

Reply:
I might sweep this (and others) up with a @grid-unit variable... no need to worry for this PR though.